src/components/TasksTable/index.jsx

```diff
@@ -253,12 +253,20 @@ export default class TasksTable extends Component {
   handleDrawerOpen = ({ target: { name } }) => {
```

Comment:
When we use `memoizeWith`, we are caching a value (the return value). The async function is doing setState but not returning any value so we are not really caching any value. Also, we should not be setting state inside `memoizeWith` because that mean this state will not be set when the value is already cached.